FWCore/ParameterSet/python/Config.py

```diff
@@ -1479,6 +1481,8 @@ def __init__(self,process, SelectEvents = untracked.PSet(), outputCommands = unt
         self.__process = process
         self.__SelectEvents = SelectEvents
         self.__outputCommands = outputCommands
+        # Need to remove MessageLogger from the subprocess now that MessageLogger is always present
+        del self.__process.MessageLogger
```

Comment:
Are the MessageLogger objects guaranteed to be the same? [Hopefully] If they are not, wouldn't we want to 'forward' extra categories to the parent?

Reply:
I added a printout of a similar warning if the `process.MessageLogger` objects differ between the parent Process and the SubProcess. If the objects are the same, all changes done via the SubProcess exist automatically already in the parent Process too.